include/envoy/http/filter.h

```diff
@@ -151,6 +151,14 @@ class StreamFilterCallbacks {
    */
   virtual Router::RouteConstSharedPtr route() PURE;
 
+  /**
+   * Schedules a request for a RouteConfiguration update from the management server.
+   * @param cb callback to be called when the configuration update has been propagated to the worker
+   * thread.
+   * @return whether RouteConfiguration update has been scheduled.
```

Comment:
Could you add to these comments something about what happens with the cb when the return value is false? It will never be called, I assume?

Reply:
done.